### PR TITLE
OIDC server config snippet updates

### PR DESCRIPTION
**Pull request by ManasiGandhi**

### CODE REVIEW

File: ref/feature/openidConnectServer-1.0.adoc [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+:linkcss:
 :page-layout: feature
-:nofooter: 
+:nofooter:
 
 include::./gen/openidConnectServer-1.0.gendoc[tag=description]
 
 include::./gen/openidConnectServer-1.0.gendoc[tag=enable]
 
+include::./gen/openidConnectServer-1.0./examples.adoc[]
```
Comment (Charlotte-Holt) [inline ref/feature/openidConnectServer-1.0.adoc]: I think this should be:
`include::./openidConnectServer/examples.adoc[]`

(Never done this before, but going off the example from https://raw.githubusercontent.com/OpenLiberty/docs/master/ref/feature/jdbc-4.0.adoc.)

Also, do you need to add the description file too?

Comment (ManasiGandhi) [reply ref/feature/openidConnectServer-1.0.adoc]: Yes. Need to add the description file but need to discuss before I do so as not sure.

File: ref/feature/openidConnectServer-1.0.adoc [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
+:linkcss:
 :page-layout: feature
-:nofooter: 
+:nofooter:
 
 include::./gen/openidConnectServer-1.0.gendoc[tag=description]
 
 include::./gen/openidConnectServer-1.0.gendoc[tag=enable]
 
+<<<<<<< Updated upstream
```
Comment (lauracowen) [inline ref/feature/openidConnectServer-1.0.adoc]: Don't think these git diff lines should still be in the file.

File: ref/feature/openidConnectServer-1.0.adoc [hunk 3]
```diff
@@ -1,12 +1,17 @@
...
+include::./openidConnectServer/examples.adoc[]
+
+=======
```
Comment (lauracowen) [inline ref/feature/openidConnectServer-1.0.adoc]: See above but think this git diff line should be removed.

File: ref/feature/openidConnectServer-1.0.adoc [hunk 4]
```diff
@@ -1,12 +1,17 @@
...
+>>>>>>> Stashed changes
```
Comment (lauracowen) [inline ref/feature/openidConnectServer-1.0.adoc]: See above but think this git diff line should be removed.

File: ref/feature/openidConnectServer-1.0.adoc (hunk 2 above)
Comment (ManasiGandhi) [reply ref/feature/openidConnectServer-1.0.adoc]: Removed.

File: ref/feature/openidConnectServer-1.0.adoc (hunk 3 above)
Comment (ManasiGandhi) [reply ref/feature/openidConnectServer-1.0.adoc]: Removed.

File: ref/feature/openidConnectServer-1.0.adoc (hunk 4 above)
Comment (ManasiGandhi) [reply ref/feature/openidConnectServer-1.0.adoc]: Removed.

File: ref/feature/openidConnectServer-1.0.adoc (hunk 2 above)
Comment (ManasiGandhi) [reply ref/feature/openidConnectServer-1.0.adoc]: Removed

**Pull request merged**